### PR TITLE
Represent Empty Values as Null If There Is No Empty Value

### PR DESCRIPTION
**Pull request by nikku**
# Changes

* if there is no empty value (number, select) empty values will be represented as `null`

---

Closes https://github.com/bpmn-io/form-js/issues/169

### CODE REVIEW

**Review by philippfromme — APPROVED** [review]
🚀

File: packages/form-js-viewer/test/spec/Form.spec.js [hunk 1]
```diff
@@ -288,6 +288,32 @@ describe('Form', function() {
   });
 
 
+  it('should render empty', async function() {
```
Comment (nikku) [inline packages/form-js-viewer/test/spec/Form.spec.js:291]: I'd love to have an integration test to verify the behavior (`null` fields being submitted). @philippfromme

Comment (nikku) [reply packages/form-js-viewer/test/spec/Form.spec.js:291]: From what I see we test unit-test wise only, not end-to-end at the moment.

Comment (philippfromme) [reply packages/form-js-viewer/test/spec/Form.spec.js:291]: I've added that test in a follow-up pull request: https://github.com/bpmn-io/form-js/pull/210/files#diff-f7f5270dd58d99d4b13f56770a7bcd44bee6b5d7a2f7d467fc4cf29b8f6d5093R314 🤡 Would that be alright?

Comment (nikku) [reply packages/form-js-viewer/test/spec/Form.spec.js:291]: Very nice :).

**Pull request merged**